src/vorta/views/exception_dialog.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import platform
+
+from PyQt6 import uic
+from PyQt6.QtWidgets import QApplication
+
+from vorta._version import __version__
+from vorta.utils import borg_compat
+from vorta.views.utils import get_colored_icon
+
+from ..utils import get_asset
+
+# Load UI file
+uifile = get_asset('UI/exceptiondialog.ui')
+ExceptionDialogUI, ExceptionDialogBase = uic.loadUiType(uifile)
+
+
+class ExceptionDetails:
+    @staticmethod
+    def get_os_details():
+        uname_result = platform.uname()
+        os_details = f"OS: {uname_result.system}\n"
+        os_details += f"Node Name: {uname_result.node}\n"
```

Comment:
Not sure if this is necessary.
Looks good for me too otherwise:

![image](https://github.com/borgbase/vorta/assets/41837037/cdffc5ac-07de-4dcf-a181-0fc75a28363d)

Reply:
Maybe use a format string for the datetime, so it doesn't put the subsecond decimals there.

About the "Log:":
- that's a traceback and guess you did not get it from the log, but directly within python.
- but, vorta (and also borg) do write a log, which might have some additional interesting infos, which you do not yet include. maybe a link that opens the correct log file so that the user can easily c&p stuff from there would be useful here.